boards/st/stm32h573i_dk/stm32h573i_dk.yaml

```diff
@@ -24,4 +24,5 @@ supported:
   - usb_device
   - usb
   - i2c
+  - sdhc
```

Comment:
nit- this tag will only enable SDHC driver tests (which won't work with the ST disk driver), so it can probably be dropped. Perhaps the `- disk` tag would be better here, so the disk tests are run?